drivers/sqljs-driver/src/jsMain/kotlin/com/squareup/sqldelight/drivers/sqljs/sqljs.SqlJs.kt

```diff
@@ -0,0 +1,68 @@
+@file:JsQualifier("SqlJs")
+@file:Suppress("INTERFACE_WITH_SUPERCLASS", "OVERRIDING_FINAL_MEMBER", "RETURN_TYPE_MISMATCH_ON_OVERRIDE", "CONFLICTING_OVERLOADS", "EXTERNAL_DELEGATION")
+package com.squareup.sqldelight.drivers.sqljs
+
+import org.khronos.webgl.Uint8Array
+import kotlin.js.*
```

Comment:
nit no * imports, you can search for star imports in intellij preferences to turn off

Reply:
Fixed